src/engine_render.h

```diff
@@ -354,12 +354,13 @@ struct MapVolumeBox { // sizeof = 24
 
 /******************************************************************************/
 // Stripey Line Color Arrays
-#define STRIPEY_LINE_COLOR_COUNT 8
+#define STRIPEY_LINE_COLOR_COUNT 9
```

Comment:
You may just add this as the last member of corresponding enum

Reply:
Done in d78dc4b.

I think there is a slight loss of "readability" with putting it in the enum; @Loobinex could you just check that you are happy with this change? I think is probably better overall to have it in the way it is after d78dc4b/ how Sim suggested.

Reply:
@SimLV can you resolve this if my commit is OK.

Reply:
Ok